bin/make/build.sh

```diff
@@ -9,12 +9,10 @@ BUILD_DIR="build"
 XC_PROJECT="iOSDeviceManager.xcodeproj"
 XC_TARGET="iOSDeviceManager"
 
-hash xcpretty 2>/dev/null
-if [ $? -eq 0 ] && [ "${XCPRETTY}" != "0" ]; then
-  XC_PIPE='xcpretty -c'
-else
+XC_PIPE="xcpretty -c"
+hash "xcpretty" 2>/dev/null && [ "${XCPRETTY}" != "0" ] || {
```

Comment:
Interesting rewrite.  I had to test it because I did not understand the last `||`